Fix JSONDecodeError on get_channel from EU countries

twissell- opened the pull request:
This fixes #36 

More info [here](https://stackoverflow.com/questions/74127649/is-there-a-way-to-skip-youtubes-before-you-continue-to-youtube-cookies-messag).

Nutzer123 commented:
hey,
it looks like there is an smarter way - i checked this for the videos

`def get_initial_data(session: requests.Session, url: str) -> str:
    url = url+"&ucbcb=1"
    response = session.get(url)
    html = response.text
    return html
`

twissell- commented:
Hey @Nutzer123!

I certainly prefer your fix better.  I've implemented it into this PR, but I've changed how the parameter is passed to the request. 

Thanks!

dermasmid commented:
thanks for this @twissell- and @Nutzer123
